packages/react-art/npm/lib/Circle.art.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
-var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
+var _extends =
+  Object.assign ||
```

Comment:
Hmm noticed this, does this assume a object.assign polyfill (can use `useBuiltIns` option if so)

Reply:
It doesn't assume it. (This is very rarely used code though and doesn't matter for majority of users. So we just checked in a compiled version.)